### PR TITLE
Add Restart command

### PR DESCRIPTION
**Pull request by ahmed-bhs**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | ye
| BC breaks?    | no
| Fixed tickets | there's no ticket  😄

### CODE REVIEW

File: src/Command/Docker/Restart.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+/**
+ * Class Restart.
+ */
+final class Restart extends DockerCommand
+{
+    protected function configure(): void
+    {
+        parent::configure();
+        $this->setName('docker:start')->setDescription('Restart all the services (or just one).');
```
Comment (Plopix) [inline src/Command/Docker/Restart.php:20]: `docker:start`? I think `docker:restart` would be better!

File: src/Command/Docker/Restart.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        $this->addArgument('service', InputArgument::OPTIONAL, 'Service to restart', '');
+        $this->setAliases(['restart']);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->dockerClient->start($input->getArgument('service'));
```
Comment (Plopix) [inline src/Command/Docker/Restart.php:27]: Same here, you should `restart`, no?

**Review by Plopix — CHANGES_REQUESTED** [review]
Hello @ahmed-bhs ! Thank you! for this! I asked for few fixes! but that will be merged!

File: src/Command/Docker/Restart.php (hunk 2 above)
Comment (ahmed-bhs) [reply src/Command/Docker/Restart.php]: done!

File: src/Command/Docker/Restart.php (hunk 1 above)
Comment (ahmed-bhs) [reply src/Command/Docker/Restart.php]: It is done. Thank you, I appreciate for the code review 🙏

**Comment by Plopix** [comment]
merged! thanks @ahmed-bhs !

**Comment by Plopix** [comment]
tagged!

**Pull request merged**